app/src/main/res/anim/next_guess_button_anim.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:fillAfter="true">
+
+    <alpha
+        android:fromAlpha="0.0"
+        android:toAlpha="1.0"
+        android:duration="1000" />
```

Comment:
This is the same alpha animation than for the score/distance text, I wonder if there is a way to factorise this. Maybe adding the values to strings.xml and then referancing them with "@string/..." would be better. I don't know if it is overkill tho.

Reply:
done